VAMobile/src/screens/HealthScreen/SecureMessaging/EditDraft/EditDraft.tsx

```diff
@@ -183,7 +183,7 @@ function EditDraft({ navigation, route }: EditDraftProps) {
       setBody(message?.body || '')
       setCategory(message?.category || '')
       setSubject(message?.subject || '')
-      setTo(message?.recipientId.toString() || '')
+      setTo(message?.recipientId?.toString() || '')
```

Comment:
This would still cause an error if `recipientId` is `undefined` since `toString` expects an instance of some type. You'll probably need to do something like:
```tsx
setTo((message?.recipientId || '').toString())
```

The [type](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/534d62d4b7961123d7b32bec2ca3b9776252e6af/VAMobile/src/api/types/SecureMessagingData.ts#L23) for this attribute should also be updated to `recipientId?: number` so that TypeScript will throw an error if this is referenced anywhere else without caution

Reply:
good catch and applied